packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -46,11 +46,14 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
 
     normalizeExtraEntryPoints(buildOptions.styles, 'styles').forEach(style => {
       let resolvedPath = path.resolve(root, style.input);
-      if (!fs.existsSync(resolvedPath)) {
+      if (fs.existsSync(resolvedPath)) {
+        resolvedPath = fs.realpathSync(resolvedPath);
```

Comment:
Can you add a check for the `preserveSymlinks` option here as well?  Always using the real path could break some use cases.